### ISSUE TITLE
Metrics dashboard

### ISSUE THREAD

**Issue by roborourke** [humanmade/aws-analytics#13]
It'd be good to show some of the metrics being recorded in the CMS admin. This could include:

- Current active sessions
- Total page views (histograms of months / weeks / days?)
- Top pages
- Top authors
- Top referrers
- Campaigns (utm)
- Top locations (depends on #12 )
- Bounce rate
- Per page
  - page views
  - avg time on page
  - avg scroll depth
  - bounce rate
  - top referrers

If we can extract this information and keep a time series recorded in the database then it paves the way for detecting changes or possibly some ML applications to generate insights and notifications.

**Comment by jennybeaumont** [comment]
For reference https://github.com/humanmade/platform-dev/issues/429

**Comment by rmccue** [comment]
Closing in favour of main Productboard one.
